tests/query_planner/ex1_qg.json

```diff
@@ -1,25 +1,25 @@
 {
-  "nodes" : {
-    "n0" : {
-      "category" : "biolink:MolecularEntity"
-    },
-    "n1" : {
-      "id" : "MONDO:0005148"
-    },
-    "n2" : {
-      "category" : "biolink:GeneOrGeneProduct"
-    }
-  },
-  "edges" : {
-    "e01" : {
-      "subject" : "n1",
-      "object" : "n0",
-      "predicate" : "biolink:treated_by"
-    },
-    "e02" : {
-      "subject" : "n0",
-      "object" : "n2",
-      "predicate" : "biolink:affects_abundance_of"
-    }
-  }
+	"nodes": {
```

Comment:
Are these tabs?

Reply:
Not as far as I can tell

Reply:
Okay they actually are tabs - I'm working on a fix